src/main/java/com/treasuredata/client/TDClient.java

```diff
@@ -993,4 +1010,16 @@ else if (!jobRequest.getResultOutput().isEmpty()) {
             return Optional.absent();
         }
     }
+
+    @Override
+    public TDImportResult importFile(String databaseName, String tableName, String format, File file)
```

Comment:
Do we need this method? I think we will not recommend using this API because it cannot check the duplicates if I understand correctly.

Reply:
It will return a unique-id. ruby-client also support both and I think most user might use this import-api without duplication check.